README.md

```diff
@@ -37,21 +34,93 @@ $ ./spin --help
 
 After you follow the [quickstart document](https://developer.fermyon.com/spin/quickstart/),
 you can follow the
-[Rust](https://developer.fermyon.com/spin/rust-components/) or [Go](https://developer.fermyon.com/spin/go-components/)
+[Rust](https://developer.fermyon.com/spin/rust-components/), [JavaScript](https://developer.fermyon.com/spin/javascript-components), [Python](https://developer.fermyon.com/spin/python-components), or [Go](https://developer.fermyon.com/spin/go-components/)
```

Comment:
Also, on line 22, as this is the code repo rather than the docs, maybe we should more clearly tease apart the "for application developers" info (which should mostly be covered by the docs) and the "for people interested in the source code" info (which might point to the contributor guides and the SIPs).